src/components/Calculator.js

```diff
@@ -0,0 +1,18 @@
+import './Calculator.css';
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
keeping this component  as `App` is a convention and people usually look for `App` as entry point. [OPTIONAL]